File: QuanLyHienMauApi/QuanLyHienMauApi/Models/NguoiDung.cs
using System.ComponentModel.DataAnnotations;

namespace QuanLyHienMauApi.Models
{
    public class NguoiDung
    {
        [Key]
        public int ID { get; set; }

        [Required]
        [StringLength(100)]
        public string HoTen { get; set; }

        [Required]
        [StringLength(15)]
        public string SDT { get; set; }

        [Required] [StringLength(15)]
        public string MatKhau { get; set; }

        [Required]
        [StringLength(100)]
        public string Email { get; set; }

        public string? DiaChi { get; set; }
        public double? ViDo { get; set; }
        public double? KinhDo { get; set; }

        public int LoaiTaiKhoan { get; set; } = 0; // 0: CaNhan, 1: CoSoYTe, 2: Admin
        public int TrangThai { get; set; } = 0;

        public string? TokenFCM { get; set; }

        public virtual CaNhan? CaNhan { get; set; }
        public virtual CoSoYTe? CoSoYTe { get; set; }
    }
}

File: HienMau/app/src/main/java/com/example/hienmau/models/NguoiDung.java
package com.example.hienmau.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class NguoiDung implements Serializable {
    @SerializedName("ID")
    public int id;
    @SerializedName("HoTen")
    public String hoTen;

    @SerializedName("SDT")
    public String sdt;

    @SerializedName("Email")
    public String email;

    @SerializedName("DiaChi")
    public String diaChi;

    @SerializedName("ViDo")
    public Double viDo;

    @SerializedName("KinhDo")
    public Double kinhDo;

    @SerializedName("LoaiTaiKhoan")
    public int loaiTaiKhoan;
    @SerializedName("CaNhan")
    public CaNhan caNhan;
}
